### PR TITLE
[enterprise-4.5] Added note per QE

### PR DESCRIPTION
**Pull request by sagidlow**
Applies to 4.5
Pending to QE approval: @miabbott or @bgilbert 
BZ Link: https://bugzilla.redhat.com/show_bug.cgi?id=1834830

### CODE REVIEW

File: modules/installation-user-infra-machines-pxe.adoc [hunk 1]
```diff
@@ -139,6 +139,11 @@ file on your TFTP server. The `coreos.inst.image_url` parameter value is the
 location of the compressed metal RAW image on your HTTP server, and the
 `coreos.inst.ignition_url` parameter value is the location of the bootstrap
 Ignition config file on your HTTP server.
++
+[NOTE]
+====
+You can add use serial console by appending `console=ttyS0`.
```
Comment (ahardin-rh) [inline modules/installation-user-infra-machines-pxe.adoc:145]: You can use serial console

Comment (sagidlow) [reply modules/installation-user-infra-machines-pxe.adoc]: Updated!

**Comment by miabbott** [comment]
Let's restate this along these lines:

> If you need serial console access, you should append the correct `console=` setting for your hardware platform.

(`console=ttyS0` will work for most users , but some may need a different setting)

**Comment by sagidlow** [comment]
Updated the note per @miabbott comment. Waiting on @bgilbert to sign off on the new changes.

**Comment by bgilbert** [comment]
I just left https://github.com/openshift/openshift-docs/pull/29525#discussion_r577301245 on the 4.4 PR, throwing some additional words at the new NOTE.  Also note that we should remove `console=tty0` and `console=ttyS0` arguments from the default configuration.

**Comment by sagidlow** [comment]
@bgilbert - I updated per your request and comment. Please verify the changes.

**Comment by miabbott** [comment]
There are some minor changes that I suggested on https://github.com/openshift/openshift-docs/pull/29525 which are applicable here as well.

**Comment by bgilbert** [comment]
Left comments in https://github.com/openshift/openshift-docs/pull/29525#pullrequestreview-594515289.

File: modules/installation-user-infra-machines-pxe.adoc [hunk 2]
```diff
@@ -139,6 +139,11 @@ file on your TFTP server. The `coreos.inst.image_url` parameter value is the
...
 `coreos.inst.ignition_url` parameter value is the location of the bootstrap
 Ignition config file on your HTTP server.
++
+[NOTE]
+====
+This configuration does not enable serial console access on machines with a graphical console.  To configure a different console, add one or more `console=` arguments to the `APPEND` line.  For example, add `console=tty0 console=ttyS0` to set the first PC serial port as the primary console and the graphical console as a secondary console.  For more information, see link:https://access.redhat.com/articles/7212[How does one set up a serial terminal and/or console in Red Hat Enterprise Linux?].
+====
 
 ifndef::only-pxe[]
 ** For iPXE:
```
Comment (bgilbert) [inline modules/installation-user-infra-machines-pxe.adoc:149]: Just noticed that the `kernel` line below still has the `console=` arguments.

**Review by bgilbert — APPROVED** [review]

**Pull request merged**